unlock-app/src/stories/interface/FullScreenModals.stories.js

```diff
@@ -1,13 +1,27 @@
 import React from 'react'
 import { storiesOf } from '@storybook/react'
+import { Provider } from 'react-redux'
 import { FullScreenModal } from '../../components/interface/FullScreenModals'
 import {
   WalletCheck,
   PasswordPrompt,
+  ResetPasswordPrompt,
 } from '../../components/interface/modal-templates'
 import { KindOfModal } from '../../unlockTypes'
 import doNothing from '../../utils/doNothing'
 
+const emailState = {
+  userDetails: {
+    email: 'geoff@bitconnect.gov',
```

Comment:
Who's this guy????

Reply:
I keep seeing him so now I have to ask!

Reply:
A fictional character from the Department of Bitconnect!